### ISSUE TITLE
Installation trouble shooting

### ISSUE THREAD

**Issue by bramvandijk88** [palaeoware/revosim#10]
> I'm afraid I am stuck at installation. First I attempted Ubuntu, but it appears that either (i) archive.ubuntu is offline, or (ii) this domain is blocked by our institute. I've sent an email to IT, and I'll keep you updated.

Whoops, seems like it's been a while since I last run sudo apt-get update. Problem above is solved. Perhaps mention this is a possible extra step that needs to be taken (sudo apt-get update). 

Also, perhaps still address the windows issue below, because I don't think the current explanation is thorough enough. In the documentation it even says "follow standard QT protocol" which many users (me) will not be familiar with. 

 I tried the windows protocol, since I also have access to that OS. But having arrived at step 8, I have no clue what to do next. (the explanation of step 8 is quite lacking in details). QT is installed using MYSYS2. Now what?

**Comment by RussellGarwood** [comment]
Thanks for these suggestions - and for taking the time to review our paper/software! Good shout on both points, I have:
1. Updated the Linux build instructions to include an update command at the start (though why this was causing the error messages you got is a new one on me)
2. Updating the Windows instructions. I don't currently have a windows machine to hand, so I hope what I have added, based on online documentation and memory covers what is required. I would welcome any further suggested additions, however, if there is anything that is not obvious!
